pkg/chain/ethereum/ethereum.go

```diff
@@ -86,6 +84,25 @@ func (ec *EthereumChain) OnKeepClosed(
 	)
 }
 
+// OnKeepTerminated is a callback that is invoked on-chain when keep is terminated.
```

Comment:
This doc doesn't match the one in `chain.go`.

Reply:
I'd prefer to not have any function docs in `ethereum.go` and `local.go`. Everything is already documented in `chain.go`. Don't want to delay this PR so I'll just update them but I think this should be addressed in a separate PR where we'll clear up whole `ethereum.go` and `local.go`.

Reply:
Please see https://github.com/keep-network/keep-ecdsa/pull/333/commits/26fb21ef3cf006eb37e786737ba2cf92d117d53a